action.yml

```diff
@@ -14,39 +14,59 @@ inputs:
   studio:
     description: Studio name
     required: true
+  plugin_path:
+    description: Plugin Path
+    required: false
+    default: '.'    
 
 runs:
   using: "composite"
   steps:
-    - name: Download STK
+    - name: Download STK CLI
       shell: bash
-      run: curl -O https://stk.stackspot.com/installer/linux/stk.deb
+      run: |
+        curl \
+          --fail \
+          --http2-prior-knowledge \
+          --location \
+          --output /tmp/stk.deb \
+          --silent \
+          --show-error \
+          --tlsv1.3 \
+          https://stk.stackspot.com/installer/linux/stk.deb
+        
 
-    - name: Install STK
+    - name: Install STK CLI
       shell: bash
       run: |
-        sudo dpkg --install stk.deb
-        mv $HOME/.stk/bin/* /usr/local/bin
-        stk --version
+        sudo dpkg --install /tmp/stk.deb || echo installed
+        rm --force /tmp/stk.deb
+
+    - name: Show STK CLI version
+      shell: bash
+      run: $HOME/.stk/bin/stk --version
```

Comment:
Wouldn't it be better to move the `$HOME/.stk/bin/` folder instead of running ```$HOME/.stk/bin/stk``` every time you need to use stk?